tests/integration/api/relayer.rs

```diff
@@ -0,0 +1,266 @@
+//! Relayer API integration tests
+
+use crate::integration::common::{
+    client::{CreateRelayerRequest, RelayerClient},
+    logging::init_test_logging,
+    network_selection::get_test_networks,
+    registry::TestRegistry,
+};
+use openzeppelin_relayer::models::relayer::RelayerNetworkType;
+use serial_test::serial;
+use tracing::{debug, info, info_span, warn};
+
+/// Test creating and getting relayer details
+#[tokio::test]
+#[ignore = "Requires running relayer and funded signer"]
+#[serial]
+async fn test_relayer_crud() {
+    init_test_logging();
+
+    let _span = info_span!("test_relayer_crud").entered();
+
+    let networks = get_test_networks().expect("Failed to get test networks");
+
+    if networks.is_empty() {
+        panic!("No networks selected for testing");
+    }
+
+    let registry = TestRegistry::load().expect("Failed to load test registry");
+
+    let network = networks
```

Comment:
This one is repetitive and big. Let's try to optimise so test boilerplate and setup is very simple and short.